scripts/translate_locales.py

```diff
@@ -56,7 +56,7 @@ def translate_text(text, target_language, keys_context=None):
         system_prompt += "\n\nHere is additional context for some of the keys to help with translation:\n"
         for key, context in keys_context.items():
             if (
-                key in text
+                    key in text
```

Comment:
👀

Reply:
Reverted, my IDE didn't take the setup for python in this project.